ECommerce/src/main/java/com/codenation/ecommerce/Helpers/AddProductHelper.java

```diff
@@ -0,0 +1,84 @@
+package com.codenation.ecommerce.Helpers;
+
+import com.codenation.ecommerce.Pojos.AddProduct;
+import com.codenation.ecommerce.Pojos.StatusEnum;
+import com.codenation.ecommerce.PrimaryKey.OrderDetailsPrimaryKey;
+import com.codenation.ecommerce.Repository.*;
+import com.codenation.ecommerce.models.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Ferooz on 10/07/16.
+ */
+@Component
+public class AddProductHelper {
+
+    @Autowired
+    private OrdersRepository ordersRepository;
+    @Autowired
+    OrderDetailsRepository orderDetailsRepository;
+    @Autowired
+    ProductRepository productRepository;
+    @Autowired
+    InventoryRepository inventoryRepository;
+
+
+    public AddProductHelper() {
+
+    }
+
+    public ResponseEntity<?> addProduct(AddProduct addProduct, int orderId) {
```

Comment:
Log at beginning and exit points of function.
Transaction Mgmt.
Exception Handling.
See common review comments at XLS https://docs.google.com/spreadsheets/d/1hqu6WtRwc5FxdsyWuhOV1RY0LfSTTLluMXByBHWqCBY/edit#gid=56856103